src/main/resources/avro/alleleAnnotationmethods.avdl

```diff
@@ -0,0 +1,244 @@
+  Only return variants with reference alleles on the reference with this ID. One
+  of this field or `referenceName` or `features` is required.
+  */
+  union { null, string } referenceId = null;
+
+  /**
+  Required if referenceName or referenceId supplied.
+  The beginning of the window (0-based, inclusive) for which variants with
+  overlapping reference alleles should be returned.
+  Genomic positions are non-negative integers less than reference length.
+  Requests spanning the join of circular genomes are represented as
+  two requests one on each side of the join (position 0).
+  */
+  long start;
+
+  /**
+  Required if referenceName or referenceId supplied.
+  The end of the window (0-based, exclusive) for which variants with
+  overlapping reference alleles should be returned.
+  */
+  long end;
+
+
+  /**
+  Only return variant annotations for any of these features
+  If null, return all variant annotations in specified window.
+  */
+  union { null, array<org.ga4gh.models.Feature> } features;
+
+  /**
+  Only return variant annotations including these effects (SO terms).
+  If null, return all variant annotations.
+  */
+  union { null, array<org.ga4gh.models.OntologyTerm> } effects = null;
+
+
+  /**
+  Specifies the maximum number of results to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  To get the next page of results, set this parameter to the value of
+  `nextPageToken` from the previous response.
+  */
+  union { null, string } pageToken = null;
+}
+
+
+/** This is the response from `POST /variantannotations/search` expressed as JSON. */
+record SearchVariantAnnotationsResponse {
+  /**
+  The list of matching variant annotations.
+  */
+  array<org.ga4gh.models.VariantAnnotation> variantAnnotations = [];
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  Provide this value in a subsequent request to return the next page of
+  results. This field will be empty if there aren't any additional results.
+  */
+  union { null, string } nextPageToken = null;
+}
+
+/**
+Gets a list of `VariantAnnotations` matching the search criteria.
+
+`POST /variantannotations/search` must accept a JSON version of `SearchVariantAnnotationsRequest`
+as the post body and will return a JSON version of `SearchVariantAnnotationsResponse`.
+*/
+SearchVariantAnnotationsResponse searchVariantAnnotations(
+  /** This request maps to the body of `POST /variantannotations/search` as JSON. */
+  SearchVariantAnnotationsRequest request) throws GAException;
+
+
+
+/******************  /alleleannotations/search  *********************/
+/** This request maps to the body of `POST /alleleannotations/search` as JSON. */
+record SearchAlleleAnnotationsRequest {
+  /** Required. The IDs of the annotation sets to search over. */
+  array<string> annotationSetIds = [];
```

Comment:
Per #253 this should now *maybe* be singular. Though this one is definitely higher value..